src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ClassTypeParameterNameCheck.java

```diff
@@ -39,22 +39,45 @@
  * <pre>
  * &lt;module name="ClassTypeParameterName"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class MyClass1&lt;T&gt; {}        // OK
+ * class MyClass2&lt;t&gt; {}        // violation
+ * class MyClass3&lt;abc&gt; {}      // violation
+ * class MyClass4&lt;LISTENER&gt; {} // violation
+ * class MyClass5&lt;RequestT&gt; {} // violation
+ * </pre>
+ * <p>
+ * To configure the check for names that are uppercase word:
+ * </p>
+ * <pre>
+ * &lt;module name="ClassTypeParameterName"&gt;
+ *   &lt;property name="format" value="^[A-Z]{2,}$"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class MyClass1&lt;T&gt; {}        // violation
+ * class MyClass2&lt;t&gt; {}        // violation
+ * class MyClass3&lt;abc&gt; {}      // violation
+ * class MyClass4&lt;LISTENER&gt; {} // OK
+ * class MyClass5&lt;RequestT&gt; {} // violation
+ * </pre>
  * <p>
- * An example of how to configure the check for names that are only a single
- * letter is:
+ * To configure the check for names that are camel case word with T as suffix:
```

Comment:
`... with T as suffix (<href="https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s5.2.8-type-variable-names">Goolge Style</a>)`

Reply:
we ask to generate web site for reason to let us clearly see final rendering.
if you review it yourself, you will see that wording of this like become weird to read, because you missed `(` that I suggested.
please do.